src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -305,6 +306,25 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
           pageId,
         );
         break;
+      case 'event': {
+        let addToCalendar;
+        for (const icsFile of icsFiles) {
+          if (
+            page.fieldAddToCalendar !== null &&
+            icsFile.fid === parseInt(page.fieldAddToCalendar.fileref, 10)
+          ) {
+            addToCalendar = icsFile.url;
```

Comment:
I'm not sure I understand this exactly, but are we only linking to one ics file here, while there could be multiple of them?

Reply:
No there will only be one per event - it's generated with info from the event and then referenced via a file id on the event itself in fieldAddToCalendar. Here I'm using that id to find the actual file to be downloaded and then saving that file's url in a variable that i can print out on the template.